src/main/java/com/cema/users/controllers/UserController.java

```diff
@@ -48,11 +49,45 @@
     private final CemaUserRepository cemaUserRepository;
     private final UserMapping userMapping;
     private final AuthorizationService authorizationService;
+    private final UserValidationService userValidationService;
 
-    public UserController(CemaUserRepository cemaUserRepository, UserMapping userMapping, AuthorizationService authorizationService) {
+    public UserController(CemaUserRepository cemaUserRepository, UserMapping userMapping,
+                          AuthorizationService authorizationService, UserValidationService userValidationService) {
         this.cemaUserRepository = cemaUserRepository;
         this.userMapping = userMapping;
         this.authorizationService = authorizationService;
+        this.userValidationService = userValidationService;
+    }
+
+    @ApiOperation(value = "Retrieve a user data by username", response = User.class)
```

Comment:
is this the real description of the method?

Reply:
fixed